pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/ImageVenueUploaderSection/ImageVenueUploaderSection.tsx

```diff
@@ -18,12 +18,16 @@ import { VenueImageUploaderModal } from './VenueImageUploaderModal'
 type Props = {
```

Comment:
Toutes les interface de props sont nommé comme suit : 

```
interface I<ComponentName>Props
```

ici : 
```suggestion
type IImageVenueUploaderSectionProps = {
```